app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -97,24 +145,51 @@ import com.duckduckgo.app.widget.ui.AddWidgetInstructionsActivity
 import com.duckduckgo.widget.SearchWidgetLight
 import com.google.android.material.snackbar.Snackbar
 import dagger.android.support.AndroidSupportInjection
-import kotlinx.android.synthetic.main.fragment_browser_tab.*
-import kotlinx.android.synthetic.main.include_add_widget_instruction_buttons.view.*
-import kotlinx.android.synthetic.main.include_cta_buttons.view.*
-import kotlinx.android.synthetic.main.include_dax_dialog_cta.*
-import kotlinx.android.synthetic.main.include_find_in_page.*
-import kotlinx.android.synthetic.main.include_new_browser_tab.*
+import kotlinx.android.synthetic.main.fragment_browser_tab.autoCompleteSuggestionsList
```

Comment:
I've removed any formatting from my side now. I was worried it would be `ktlint` formatting but we explicitly exclude `no-wildcard-imports` from being formatted.